packages/store/src/lib/StoreSchema.ts

```diff
@@ -34,10 +43,47 @@ export interface SerializedSchema {
 	>
 }
 
+/** @public */
+export interface SerializedSchemaV2 {
+	schemaVersion: 2
+	sequences: {
+		[sequenceId: string]: {
+			version: number
+			retroactive: boolean
+		}
```

Comment:
Why do we store `retroactive` here? afaict, we only ever read this property off of the migrations sequence in the code, not this stored one

Reply:
it is used in `compareSchemas` but I'm now considering removing that because it's not really reliable any more (see pr description for details). so I might be able to just have the version number here.

Reply:
removed